oop_q3.py

```diff
@@ -0,0 +1,67 @@
+from abc import ABC, abstractmethod
+from math import pi
+
+
+class Shape(ABC):
+    """An abstract class to define a shape. It contains the abstract methods
+    parameter and area."""
+
+    @abstractmethod
+    def get_parameter(self) -> float:
+        """An abstract method to return the parameter of the shape."""
+        pass
+
+    @abstractmethod
+    def get_area(self) -> float:
+        """An abstract method to return the area of the shape."""
+        pass
+
+
+class Rectange(Shape):
+    """A class that represents a Rectangle Shape. It implements the abstract
+    class "Shape" and it's methods get_parameter and get_area."""
+
+    def __init__(self, length: float, width: float) -> None:
+        super().__init__()
+        self.length = length
+        self.width = width
+
+    def get_parameter(self) -> float:
+        """Returns the parameter of the rectangle."""
+
+        return self.length + self.width
```

Comment:
nit: Shouldn't this be
```suggestion
        return 2 * (self.length + self.width)
```

Reply:
Thanks for the correction, guess geometry is not my main strength :p